conf/ebi_codon.config

```diff
@@ -14,15 +14,15 @@ params {
 
     antismash_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/antismash/7.1.x"
 
-    dbcan_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/dbcan/4.0"
+    dbcan_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/dbcan/4.1.3-V12"
```

Comment:
We need to update the auto-download bit and uplod this version to the FTP

Reply:
It is on FTP

Reply:
Perfect, I'll update this while doing the end-to-end test case